src/components/OasisMarketWidget.js

```diff
@@ -1,16 +1,82 @@
-import React, { PureComponent } from 'react'
+import React, { PureComponent } from 'react';
 import { PropTypes } from 'prop-types';
+import { last } from 'lodash';
+
 import OasisWidgetFrame from '../containers/OasisWidgetFrame';
+import { OasisTable } from './OasisTable';
+import { ETH_UNIT_ETHER } from '../constants';
+import { TIME_SPAN_DAY, TIME_SPAN_MONTH, TIME_SPAN_WEEK } from '../utils/timeSpan';
+import getTokenPairPrice from '../utils/tokens/getTokenPairPrice';
+import getTokenPairVolume from '../utils/tokens/getTokenPairVolume';
```

Comment:
Why don't we have those in a single file o.O ?

Reply:
Good point, i  think we can move this to one file

Reply:
done